### PR TITLE
Labels and owners list

### PR DESCRIPTION
**Pull request by elviraux**
This yaml file lists labels and owners for kubeflow/kubeflow repo
original doc https://docs.google.com/spreadsheets/d/1TL0ywFxCwz25uUDzviguxRQfVE1i4C7IYFADXswwWRg/edit#gid=0

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/296)

### CODE REVIEW

File: labels-owners.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            - rakelkar
+            - yuzisun
+        area/istio:
+          owners:
+            - krishnadurai
+            - kunmingg
+            - lluunn
+        area/jupyter:
+          owners:
+            - avdaredevil
```
Comment (jlewi) [inline labels-owners.yaml]: remove avdaredevil, prodonjs, ioandr, pdmack and llun

File: labels-owners.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          owners:
+            - alsrgv
+            - jiezhang
+            - jlewi
+        area/openvino:
+          owners:
+            - kkasravi
+        area/pipelines:
+          owners:
+            - ark-kun
```
Comment (jlewi) [inline labels-owners.yaml]: add jessiezcc

File: labels-owners.yaml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            - jlewi
+            - kunmingg
+            - lluunn
+            - richardsliu
+            - zhenghuiwang
+        area/tfjob:
+          owners:
+            - gaocegege
+            - johnugeorge
+            - ichardsliu
```
Comment (jlewi) [inline labels-owners.yaml]: Richard's github id is wrong.

File: labels-owners.yaml [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+            - scorpioCPH
```
Comment (jlewi) [inline labels-owners.yaml]: remove scorpioCPH

File: labels-owners.yaml (hunk 3 above)
Comment (jlewi) [reply labels-owners.yaml]: Remove @gaocegege its mostly  @richardsliu

File: labels-owners.yaml [hunk 5]
```diff
@@ -0,0 +1,157 @@
+
+# This file provides information about labels and owners behind them
+#
+# Please keep this file alphabetized by label name and owner name
+
+repositories:
```
Comment (jlewi) [inline labels-owners.yaml]: Lets not group by repositories and kubeflow just remove those levels.

We should be applying these labels consistently.

**Comment by jlewi** [comment]
Thanks @elviraux Could you also update the triage doc to reference this file?

File: labels-owners.yaml [hunk 6]
```diff
@@ -0,0 +1,157 @@
+
```
Comment (gaocegege) [inline labels-owners.yaml]: Do we have labels for automl (katib), pytorch, xgboost and so on?

File: labels-owners.yaml (hunk 3 above)
Comment (gaocegege) [reply labels-owners.yaml]: I can help maintain the tf-operator while I am more interested in these unmatured operators like xgboost, mxnet, mpi, and others.

**Comment by elviraux** [comment]
> Thanks @elviraux Could you also update the triage doc to reference this file?

sure

File: labels-owners.yaml (hunk 6 above)
Comment (elviraux) [reply labels-owners.yaml]: added

File: labels-owners.yaml [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    owners:
+      - krishnadurai
+      - kunmingg
+      - lluunn
+  area/jupyter:
+    owners:
+      - kimwnasptd
+      - vkoukis
+  area/katib:
+      - andreyvelich
```
Comment (johnugeorge) [inline labels-owners.yaml]: Please add @gaocegege in Katib

File: proposals/issue_triage.md [hunk 8]
```diff
@@ -26,7 +26,7 @@ The following are necessary and sufficient conditions for an issue to be conside
   * **process**
     * Typically used to leave a paper trail for updating Kubeflow infrastructure. It helps to track the changes to infrastructure for easy debugging in the future.
 
-* The issue must have at least one area label indicating the WG/SIG that should own it
+* The issue must have at least one [area label](https://github.com/kubeflow/community/blob/master/labels-owners.yaml) indicating the WG/SIG that should own it.
```
Comment (jlewi) [inline proposals/issue_triage.md]: nit WG/IG are not 1:1 to area/labels. Maybe just say one area label grouping related issues and relevant owners.

**Comment by jlewi** [comment]
@elviraux do you want to address my couple nitpicky comments and then we can get this merged?

**Comment by elviraux** [comment]
@jlewi sorry, no idea how i didn't get notifications. going to add changes today
